internal/pkg/rollback/rollback.go

```diff
@@ -11,30 +11,34 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// RollbackFunc is a function that is called in order to perform a rollback.
 type RollbackFunc func(ctx context.Context) error
 
 type rollbackInfo struct {
 	name string
 	fn   RollbackFunc
 }
 
+// Rollback is used to track RollbackFuncs
```

Comment:
dot "." at the end? you seemed to be adding these above

Reply:
added them everwhere